### DISCUSSION TITLE
Full-width section support for webparts

### DISCUSSION THREAD

**Discussion by faisal-imam** [microsoft-search/pnp-modern-search#2155] [Ideas]
The **PnP - Search Results** webpart supports the full-width section which is good because it means that there the unused space on the left and right is not wasted. But the other webparts i.e. **PnP - Search Box**, **PnP - Search Filters** and **PnP - Search Verticals** do not support full-width section. So if I place the Search Box and Search Filters on top of the Search Results, the page looks un-even because the Search Box and Filters are center aligned and the Search Results is not.

Can we please have full-width section support for these remaining three webparts as well?

**Comment by wobba** [comment]
Fixed in #2162
